src/app/send/add-edit.component.html

```diff
@@ -1,67 +1,130 @@
-<ng-container *ngIf="send">
-    <form (ngSubmit)="submit()" [appApiAction]="formPromise">
-        <div class="content" *ngIf="send">
-            <div class="inner-content">
-                <div class="box">
-                    <div class="box-header">
-                        {{'editSend' | i18n}}
+<form (ngSubmit)="submit()" [appApiAction]="formPromise">
+    <div class="content">
+        <div class="inner-content" *ngIf="send">
+            <div class="box">
+                <app-callout *ngIf="disableSend">
+                    <span>{{'sendDisabledWarning' | i18n}}</span>
+                </app-callout>
+            </div>
+            <div class="box">
+                <div class="box-header">
+                    {{title}}
+                </div>
+                <div class="box-content">
+                    <div class="box-content-row" appBoxRow>
+                        <label for="name">{{'name' | i18n}}</label>
+                        <input id="name" type="text" name="Name" [(ngModel)]="send.name" appAutofocus [readOnly]="disableSend">
                     </div>
-                    <div class="box-content">
-                        <div class="box-content-row" appBoxRow>
-                            <label for="name">{{'name' | i18n}}</label>
-                            <input id="name" type="text" name="Name" [(ngModel)]="send.name" appAutofocus>
+                </div>
+            </div>
+            <div class="box">
+                <div class="box-content">
+                    <ng-container *ngIf="!editMode">
+                        <div class="box-content-row box-content-row-radio">
+                            <label class="radio-header">{{'whatTypeOfSend' | i18n}}</label>
+                            <div class="item" *ngFor="let o of typeOptions">
+                                <input type="radio" class="radio" [(ngModel)]="send.type" name="Type_{{o.value}}"
+                                    id="type_{{o.value}}" [value]="o.value" (change)="typeChanged(o)"
+                                    [checked]="send.type === o.value">
+                                <label class="label-hack" for="type_{{o.value}}">
```

Comment:
Is there are more comely name we can choose for this class other than `"label-hack"`? What is the intent of this class; I see the `:not()` rule in the stylesheet, however what is it not supposed to be doing? Let's name it explicitly based on function vs. what it is (a hack) 🪓

Reply:
Unfortunately I'm not sure what that functional name would be. Maybe `.unstyled-label`? All applying this class is doing is ignoring the `<label>` styling that is forced on it just for being in a `.box-row`. Things like `display: block; width: 100%` aren't normal `<label>` styles and don't work in the context of radio buttons, but there is no way around that as long as all `<label>`s are getting that style. 

The real solution here would be to remove the default `label` stylings and use `.row-label` when we need these styles, so I guess my hope with just calling the class `label-hack` is that it will keep attention and get removed. Checking every label in the app just seemed out of scope for Send, especially for testing.

Reply:
Great thoughts, thanks. how about `unstyled` and you can put some comments in the scss that relate to the HACK and TODO nature of what you have mentioned above.

Reply:
Done!